content/tutorials/xgboost_deployment.md

```diff
@@ -4,79 +4,57 @@ date= 2018-03-12T17:08:30-07:00
 description = "Guide to deploy XGBoost Models."
 draft= false
 +++
-## A guide to installing packages to model containers
+## A guide to deploy XGBoost Models
 {{% notice warning %}}
 This tutorial will assume that you have already installed `clipper_admin` and its dependencies.
 {{% /notice %}}
 
-In order to keep the base image's light, containers limit the packages they come with default; however, Clipper also supports the installation of additional packages onto model containers using pip. Packages such as XGBoost can be installed and deployed in this way.
+In order to keep the base images small, containers limit the packages they come with default; however, Clipper also supports the installation of additional packages onto model containers using pip. Packages such as XGBoost can be installed and deployed in this way.
 
-First, we open a python interactive shell.
+First, we open a python interactive shell, and start clipper.
 ```sh
-$ python
-# Now, we start clipper.
 >>> import logging, xgboost as xgb, numpy as np
 >>> from clipper_admin import ClipperConnection, DockerContainerManager
 >>> cl = ClipperConnection(DockerContainerManager())
->>> logging.info("Starting Clipper")
-18-03-15:11:20:54 INFO     [<stdin>:1] Starting Clipper
->>> cl.start_clipper()
-18-03-15:11:21:00 INFO     [docker_container_manager.py:96] Starting managed Redis instance in Docker
-18-03-15:11:22:07 INFO     [clipper_admin.py:109] Clipper still initializing.
-18-03-15:11:22:08 INFO     [clipper_admin.py:111] Clipper is running
-# Next, we must register our application
+```
+Next, we must register our application
+```sh
 # We will register it to deploy an xgboost model.
 >>> cl.register_application('xgboost-test', 'integers', 'default_pred', 100000)
-18-03-15:11:23:17 INFO     [clipper_admin.py:186] Application xgboost-test was successfully registered
-# Now, we define a function to generate test points.
->>> def get_test_point():
-...     return [np.random.randint(255) for _ in range(784)]
-...
-# And create a training matrix
+```
+We also define a function to generate test points.
```

Comment:
Change this to "Now we need to get some training data. For the purposes of this example, we will just use randomly generated data."